docs/semgrep-ci/network-broker.md

```diff
@@ -87,9 +87,13 @@ Ensure that you are logged in to the server where you want to run Semgrep Networ
 
    ![Screenshot of Semgrep AppSec Platform's Network Broker page](/img/scp-broker.png#md-width)
 
-2. Update the `config.yaml` by replacing `YOUR_BASE_URL` with your GitLab or GitHub URL:
+2. Update the `config.yaml` by replacing `YOUR_BASE_URL` with your Bitbucket Data Center, GitLab, or GitHub URL:
```

Comment:
@vivekkhimani is it just BBDC, or does it/should it work with Bitbucket Cloud?

Reply:
Broker should only be relevant for BBDC!